src/gs-image-tile.ui

```diff
@@ -140,69 +217,20 @@
               </object>
 	    </child>
 	    <child>
-	      <object class="GtkEventBox" id="eboxbox">
-                <property name="visible">True</property>
-		<property name="halign">fill</property>
-		<property name="valign">fill</property>
-                <child>
-                  <object class="GtkBox" id="box">
-		    <property name="visible">True</property>
-		    <property name="orientation">vertical</property>
-		    <property name="spacing">6</property>
-                    <property name="margin">12</property>
-                    <property name="margin-top">100</property>
-		    <property name="halign">start</property>
-		    <property name="valign">start</property>
-		    <child>
-		      <object class="GtkLabel" id="app_name">
-                        <property name="visible">True</property>
-                        <property name="valign">end</property>
-                        <property name="xalign">0</property>
-                        <property name="ellipsize">end</property>
-                        <property name="width_chars">12</property>
-                        <property name="max_width_chars">12</property>
-			<style>
-			  <class name="app-name"/>
-			</style>
-		      </object>
-		      <packing>
-                        <property name="expand">False</property>
-                        <property name="fill">True</property>
-		      </packing>
-		    </child>
-		    <child>
-		      <object class="GtkLabel" id="app_summary">
-                        <property name="visible">True</property>
-                        <property name="valign">end</property>
-                        <property name="xalign">0</property>
-                        <property name="ellipsize">end</property>
-                        <property name="lines">2</property>
-                        <property name="wrap">True</property>
-			<property name="width_chars">12</property>
-                        <property name="max_width_chars">12</property>
-			<style>
-			  <class name="app-summary"/>
-			</style>
-		      </object>
-		      <packing>
-                        <property name="expand">False</property>
-                        <property name="fill">True</property>
-		      </packing>
-		    </child>
-		    <child>
-		      <object class="GsStarWidget" id="stars">
-                        <property name="visible">True</property>
-                        <property name="halign">center</property>
-		      </object>
-		      <packing>
-                        <property name="expand">False</property>
-                        <property name="fill">True</property>
-		      </packing>
-		    </child>
-                  </object>
-                </child>
+	      <!-- The fallback icon should be covered by the main image but
+	           is here for the cases where that image doesn't exist, as
+	           without a graphical reference it is very difficult to spot
+	           the applications -->
```

Comment:
Does e.g. Glade preserve comments in UI files like this? Otherwise it would be better to have this comment in the code somewhere.